### PR TITLE
[13.0] stock_location_bin_name migration to 13.0

### PR DESCRIPTION
**Pull request by guewen**
Migrate #713.
Requires #804, #805

### CODE REVIEW

**Review by simahawk — APPROVED** [review]

**Comment by simahawk** [comment]
needs rebase + pre-commit update

**Comment by guewen** [comment]
rebased

File: stock_location_bin_name/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    "name": "Stock Location Bin Name",
+    "version": "13.0.1.0.0",
+    "author": "BCIM, Okia, Camptocamp, Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/stock-logistics-warehouse",
+    "summary": "Compute bin stock location name automatically",
+    "category": "Stock Management",
+    "depends": ["stock_location_zone", "stock_location_position"],
+    "data": ["views/stock_location.xml"],
+    "installable": True,
+    "development_status": "Alpha",
```
Comment (rousseldenis) [inline stock_location_bin_name/__manifest__.py:15]: Still Alpha ?

**Comment by simahawk** [comment]
@guewen ping on https://github.com/OCA/stock-logistics-warehouse/pull/806#discussion_r395781919
Also, required PRs are merged. Can you clean up and make build happy? :)

**Review by jgrandguillaume — APPROVED** [review]
let's merge that

**Pull request merged**